.github/workflows/sdks.yaml

```diff
@@ -20,3 +20,4 @@ jobs:
       - run: make --directory sdks/${{matrix.name}} publish -B
         env:
           JAVA_SDK_MAVEN_PASSWORD: ${{ secrets.GITHUB_TOKEN }}
+          PYPI_API_TOKEN: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
We don't have this yet but I can add before next release. Could you use your fork and https://test.pypi.org/ to test that the entire release process works?

Reply:
@terrytangyuan I tested the release process and discovered two bugs:
- secret reference was missing `${}`
- the built full `dist` path was missing

I fixed those. During testing I had to name the package `argo-workflows-test` because I do not have permission to `argo-workflows`. Here are the results:
- [published package](https://test.pypi.org/project/argo-workflows-test/)
- [workflow publication run](https://github.com/flaviuvadan/argo-workflows/runs/4499417327?check_suite_focus=true)